backend/b2/api/types.go

```diff
@@ -65,6 +66,16 @@ func (t *Timestamp) UnmarshalJSON(data []byte) error {
 
 const versionFormat = "-v2006-01-02-150405.000"
 
+var /* const */ versionRegexp = regexp.MustCompile("-v\\d{4}-\\d{2}-\\d{2}-\\d{6}-\\d{3}")
```

Comment:
Is this `/* const */` annotation understood by some tool? Not seen it before?

Reply:
Honestly, it was the first suggestion I found on StackOverflow for a regexp constant, and I got the sense that it was more from the answerer's personal style guide, as a way to tell humans about the intent. I haven't worked with Go before so I have no stylistic opinions of my own. Is there something you'd prefer to indicate that kind of intent? Or just drop it and let the context explain it for humans?

Reply:
OK! I'd probably drop it if it doesn't have a meaning to a tool.